charts/index.yaml

```diff
@@ -1,28 +1,28 @@
 apiVersion: v1
 entries:
-  federation-v2:
+  kubefed:
   - apiVersion: v1
     created: 2019-05-06T15:30:47.795749649-07:00
     description: Kubernetes Federation V2 helm chart
     digest: 9b248eed6f021951653ee98cb255d2d3564748ac04c2484c941f619c39c5dd79
-    name: federation-v2
+    name: kubefed
     urls:
-    - https://sigs.k8s.io/kubefed/releases/download/v0.0.10/federation-v2-0.0.10.tgz
+    - https://github.com/kubernetes-sigs/federation-v2/releases/download/v0.0.10/federation-v2-0.0.10.tgz
```

Comment:
Good catch.  Looks like the links for old releases aren't changed by the rename.

cc: @pmorie